ynr/apps/uk_results/templates/uk_results/current_elections_with_no_resuts.html

```diff
@@ -10,6 +10,9 @@ <h2><a href="{{ group.grouper.get_absolute_url }}">{{ group.grouper }}</a></h2>
       <li>
         <a href="{{ ballot.get_absolute_url }}">{{ ballot.post.label }}</a>
         <a href="{% url "ballot_paper_results_form" ballot.ballot_paper_id %}" class="button tiny">Add results</a>
+        {% if ballot.results_by_bot %}
```

Comment:
`results_by_bot` is a method on the View, but here you are trying to access it on the ballot instance. See comment about changing to store the URL on the `Ballot` as that would allow you to do this sort of check